diff --git a/node_modules/@docusaurus/mdx-loader/lib/loader.js b/node_modules/@docusaurus/mdx-loader/lib/loader.js
index 5f9dce1..fdc1976 100644
--- a/node_modules/@docusaurus/mdx-loader/lib/loader.js
+++ b/node_modules/@docusaurus/mdx-loader/lib/loader.js
@@ -103,6 +103,11 @@ async function mdxLoader(fileString) {
     const filePath = this.resourcePath;
     const reqOptions = this.getOptions();
     const { frontMatter, content: contentWithTitle } = (0, utils_1.parseFrontMatter)(fileString);
+
+    const isPartial = filePath.split('/').pop()[0] === '_';
+    if (!isPartial) {
+        const ogImageURL = new URL('https://apify.com/og-image/docs-article');
+        ogImageURL.searchParams.set('title', frontMatter.title);
+        frontMatter.image ??= ogImageURL.toString();
+    }
+
     const { content, contentTitle } = (0, utils_1.parseMarkdownContentTitle)(contentWithTitle, {
         removeContentTitle: reqOptions.removeContentTitle,
     });
