diff --git a/node_modules/@docusaurus/mdx-loader/lib/loader.js b/node_modules/@docusaurus/mdx-loader/lib/loader.js
index 5f9dce1..1c0e519 100644
--- a/node_modules/@docusaurus/mdx-loader/lib/loader.js
+++ b/node_modules/@docusaurus/mdx-loader/lib/loader.js
@@ -106,6 +106,14 @@ async function mdxLoader(fileString) {
     const { content, contentTitle } = (0, utils_1.parseMarkdownContentTitle)(contentWithTitle, {
         removeContentTitle: reqOptions.removeContentTitle,
     });
+
+    const isPartial = filePath.split('/').pop()[0] === '_';
+    if (!isPartial) {
+        const ogImageURL = new URL('https://apify.com/og-image/docs-article');
+        ogImageURL.searchParams.set('title', frontMatter.title);
+        frontMatter.image ??= ogImageURL.toString();
+    }
+
     const hasFrontMatter = Object.keys(frontMatter).length > 0;
     if (!compilerCache.has(this.query)) {
         const remarkPlugins = [
